tests/foreman/cli/test_host.py

```diff
@@ -25,66 +37,497 @@ def setUp(self):
+        self.organization = self.host.organization  # pylint:disable=no-member
+        self.ptable = self.host.ptable
+        # using CLI to create host
+        self.host = Host.create({
+            u'architecture-id': self.host.architecture.id,
+            u'domain-id': self.host.domain.id,
+            u'environment-id': self.host.environment.id,
+            u'location-id': self.host.location.id,  # pylint:disable=no-member
+            u'mac': self.host.mac,
+            u'medium-id': self.host.medium.id,
+            u'name': self.host.name,
+            u'operatingsystem-id': self.host.operatingsystem.id,
+            # pylint:disable=no-member
+            u'organization-id': self.host.organization.id,
+            u'partition-table-id': self.host.ptable.id,
+            u'puppet-proxy-id': self.puppet_proxy['id'],
+            u'root-pass': self.host.root_pass,
+        })
+
+    @tier1
+    def test_positive_update_name_by_id(self):
+        """@test: A host can be updated with a new random name. Use id to
+        access the host
+
+        @feature: Hosts
+
+        @assert: A host is updated and the name matches
+        """
+        for new_name in valid_hosts_list():
+            with self.subTest(new_name):
+                Host.update({
+                    'id': self.host['id'],
+                    'new-name': new_name,
+                })
+                self.host = Host.info({'id': self.host['id']})
+                self.assertEqual(
+                    u'{0}.{1}'.format(
+                        new_name,
+                        self.host['domain'],
+                    ).lower(),
+                    self.host['name'],
+                )
+
+    @tier1
+    def test_positive_update_name_by_name(self):
+        """@test: A host can be updated with a new random name. Use name to
+        access the host
+
+        @feature: Hosts
+
+        @assert: A host is updated and the name matches
+        """
+        for new_name in valid_hosts_list():
+            with self.subTest(new_name):
+                Host.update({
+                    'name': self.host['name'],
+                    'new-name': new_name,
+                })
+                self.host = Host.info({
+                    'name': u'{0}.{1}'
+                            .format(new_name, self.host['domain']).lower()
+                })
+                self.assertEqual(
+                    u'{0}.{1}'.format(
+                        new_name,
+                        self.host['domain'],
+                    ).lower(),
+                    self.host['name'],
+                )
+
+    @tier1
+    def test_positive_update_mac_by_id(self):
+        """@test: A host can be updated with a new random MAC address. Use id
+        to access the host
+
+        @feature: Hosts
+
+        @assert: A host is updated and the MAC address matches
+        """
+        new_mac = gen_mac()
+        self.assertNotEqual(self.host['mac'], new_mac)
```

Comment:
That is pretty redundant as we never verify things like that and basically code like that doesn't verify anything